torch/_refs/nn/functional/__init__.py

```diff
@@ -45,3 +80,78 @@ def elu(
         rhs = refs.expm1(a)
 
     return refs.where(refs.gt(a, 0), a, rhs)
+
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.OP_MATH,
+)
+def selu(
+    a: TensorLikeType, inplace: bool = False
+) -> TensorLikeType:
+    """
+    Reference implementation of torch.nn.functional.selu
+    """
+
+    if inplace:
+        raise NotImplementedError
+
+    alpha = 1.6732632423543772848170429916717
+    scale = 1.0507009873554804934193349852946
+
+    python_type = utils.dtype_to_type(a.dtype)
```

Comment:
This is probably not what we want to do for alpha. If we want to verify the input is a floating dtype or higher then we should just check for that directly with utils.is_float_dtype and utils._is_complex_dtype. But we can probably avoid the type checking completely here